tests/Functional/Controller/Admin/ActivityControllerTest.php

```diff
@@ -5,6 +5,10 @@
 use App\Controller\Admin\ActivityController;
 use App\Log\EventService;
 use App\Tests\AuthWebTestCase;
+use App\Tests\Database\Activity\ActivityFixture;
```

Comment:
why did you touch this file? I see no real changes except for the fixtures.
were they needed to run the previously written tests?

Reply:
Thats really odd, Because it I added additional things to this file that required this line of code: loading in the fixtures. But not yet neccessary in this test. So I removed it.

Reply:
yeah, I saw the fixtures as well. just didn't see you using them anywhere. should have been a tad more specific with my initial comment.